javaScriptLoops-forLoop/practiceProblems/sampleBreak.js

```diff
@@ -0,0 +1,12 @@
+let message = "supercalifragilisticexpialidocious";
+message = message.length;
+
+for ( let i = 1; i < message; i++ ) {
+  if ( i === message / 2 ) {
+    console.log('The loop has terminated...');
+    break;
+  }
+  console.log(`Logging the number ${i}`);
+}
```

Comment:
There are inconsistencies in the spacing of this code compared to other files for loops and if statements. In particular, there are spaces after the opening parenthesis and before the closing parenthesis. To ensure consistency across all files, we should remove these spaces.